pkgs/by-name/gi/git-instafix/package.nix

```diff
@@ -0,0 +1,48 @@
+{ git
+, lib
+, libgit2
+, makeWrapper
+, rustPlatform
+, fetchFromGitHub
+,
+}:
+let
+  inherit
+    (lib)
+    licenses
+    maintainers
+    ;
+
+  version = "0.2.0";
+in
+rustPlatform.buildRustPackage {
+  pname = "git-instafix";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "quodlibetor";
+    repo = "git-instafix";
+    rev = "v${version}";
+    hash = "sha256-jNmgXxe0omS085/saoTLnUAqBkg2d8t6ZldeGJUGh00=";
+  };
+
+  cargoHash = "sha256-3l400AUoDmYRQQWU4VtUS/PJHoJ/Y0RGSy4N3nnH2ok=";
+
+  meta = {
+    description = "Quickly fix up an old commit using your currently-staged changes.";
+    homepage = "https://github.com/quodlibetor/git-instafix";
+    license = with licenses; [ mit asl20 ];
+    maintainers = with maintainers; [ mightyiam ];
+    changelog = "https://github.com/quodlibetor/git-instafix/releases/tag/v${version}";
+  };
```

Comment:
`meta` should be at the very bottom, at least that's what I see in most packages.